presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -118,6 +126,12 @@ private PlanNodeStatsSummarizer() {}
                             (map1, map2) -> mergeMaps(map1, map2, OperatorHashCollisionsStats::merge));
                 }
 
+                // The only statistics we have for Window Functions are very low level, thus displayed only in VERBOSE mode
+                if (operatorStats.getInfo() instanceof WindowOperator.WindowInfo && verbose) {
```

Comment:
This is also super weird - verbose should control what stats are *displayed*, not what stats are available. What if there were actually some stats about WindowOperator in both categories? It seems like verbosity should only be a factor during printing, and not otherwise. Please change the first commit to reflect this.